packages/lib-utils/src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useEventListener } from './useEventListener';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const tryJSONParse = <T = any>(data: string): string | T => {
```

Comment:
@rhamilto I think the way in which we are calling this function means we do not need the generic to have a default of `any` here.

So we'd just have:
```ts
const tryJSONParse = <T>(data: string): string | T => {
  try {
    return JSON.parse(data);
  } catch {
    return data;
  }
};
```

... since below useLocalStorage expects `T` to extend `object`

Reply:
Updated.